run.sh

```diff
@@ -0,0 +1,7 @@
+if [ ! -d savedata ]; then mkdir savedata; fi
```

Comment:
```suggestion
#!/usr/bin/env bash

if [ ! -d savedata ]; then mkdir savedata; fi
```
Add a shebang so the file can be executed directly